From 0f5b4cefff47e2ecf558a28818e1688414557a5e Mon Sep 17 00:00:00 2001
From: Prarit Bhargava <prarit@redhat.com>
Date: Fri, 31 Oct 2014 13:10:14 -0400
Subject: [x86] setup: Mark Intel Haswell ULT as supported

Message-id: <1414761014-16418-1-git-send-email-prarit@redhat.com>
Patchwork-id: 98601
O-Subject: [RHEL7.1 PATCH BZ 1159006] Mark Intel Haswell ULT as supported
Bugzilla: 1159006
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Don Zickus <dzickus@redhat.com>

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1159006
Build info: It's Halloween.  There's no building on Halloween.  Only candy.

Mark Intel Haswell ULT (6,69) as supported.  All the little bits should be in
the kernel already.

Cc: Tony Camuso <tcamuso@redhat.com>
Cc: Don Zickus <dzickus@redhat.com>
Cc: Vivek Goyal <vgoyal@redhat.com>
Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/arch/x86/kernel/setup.c b/arch/x86/kernel/setup.c
index 8af4b74..6445e02 100644
--- a/arch/x86/kernel/setup.c
+++ b/arch/x86/kernel/setup.c
@@ -870,6 +870,7 @@ static void rh_check_supported(void)
   switch (boot_cpu_data.x86_model) {
   case 77: /* Atom Avoton */
   case 70: /* Crystal Well */
+  case 69: /* Haswell ULT */
    break;
   default:
    if (boot_cpu_data.x86_model > 63) {
-- 
1.7.1